File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("remote-jobs-api-rss-feed_sdk")

describe("RemoteJobsApiRssFeedSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# RemoteJobsApiRssFeed SDK exists test

require "minitest/autorun"
require_relative "../RemoteJobsApiRssFeed_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RemoteJobsApiRssFeedSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
